docs/coach.rst

```diff
@@ -3,39 +3,39 @@
 Coach your Learners in Kolibri
 ##############################
  
-You can track progress of the **Learner** users, create and assign **Lessons** and **Exams** to classes or learner groups from the **Coach** dashboard. The default view of the **Coach** dashboard presents the list of **Classes** with number of learners enrolled to each class.
+You can track progress of the **Learners**, create and assign **Lessons** and **Exams** to classes or learner groups from the **Coach** dashboard. The default view of the **Coach** dashboard presents the list of **Classes** with their assigned coaches, and the number of learners enrolled in each class.
 
-Select a class from the list to access the progress-tracking features and create exams.
+Click on a class to access the progress-tracking features and create lessons and exams.
 
-	.. figure:: img/coach-home.png
-	  :alt: default coach view with list of classes
+.. figure:: img/coach-home.png
+  :alt: default coach view with list of classes
 
 .. _track_progress:
 
 
 View Learner Reports
 ~~~~~~~~~~~~~~~~~~~~
 
-This is the default view when you select a class from the **Coach** dashboard. It displays the list of learners in the class and the group they are asigned to.
+Clicking a class on the **Coach** dashboard displays the list of learners in the class and the group they are assigned to.
 
-When you click the username of a learner, you will see the list of content they accessed and when were they last active on each of the channels:
+When you click the username of a learner, you will see a list of content which the learner has accessed. This displays the last time they were active on each of the channels:
 
-	.. figure:: img/learner-report-content.png
-	  :alt: Learner Reports present a quick view of all the content accessed by a learner.
-
-	  Learner Reports present a quick view of all the content accessed by a learner.
+.. figure:: img/learner-report-content.png
+  :alt: Learner Reports present a quick view of all the content accessed by a learner.
 
+  Learner Reports present a quick view of all the content accessed by a learner.
 
 .. tip::
-  Username of the selected learner will be the last item in the breadcrumb menu on the top.
+  In a Learner report, the username of the selected learner will be the last item in the breadcrumb navigation menu.
```

Comment:
Since the introduction of the learner name inside the breadcrumb menu is a new feature, my fear was that it could pass as unperceived by the coach user, hence the tip and the highlighted portion of the screenshot.

Reply:
Changed to `note` instead of the `tip`.